src/templates/tasks/new.template.html

```diff
@@ -382,4 +383,37 @@ <h2>New task</h2>
     </div>
   </div>
 </form>
+<script src="https://cdn.jsdelivr.net/gh/shivanraptor/optparse-js@1.1.1/lib/optparse.js"></script>
+<script src="https://cdn.jsdelivr.net/gh/shivanraptor/optparse-js@1.1.1/lib/optparse.hashtopolis.js"></script>
```

Comment:
external library might need to be made internal?

Reply:
Right, the author said so too. I'm going to update the pull request.

Reply:
Added the changes at commit 5e4b955: https://github.com/s3inlc/hashtopolis/pull/672/commits/5e4b955f4466bd9889ac7440cc16ff18763711d5